### PR TITLE
feat: splitting fossa-scan into two jobs

### PR DESCRIPTION
**Pull request by uoboda-splunk**

### CODE REVIEW

File: .github/workflows/reusable-build-test-release.yml [hunk 1]
```diff
@@ -204,8 +203,18 @@ jobs:
         with:
           name: THIRDPARTY
           path: /tmp/THIRDPARTY
+
+  fossa-test:
+    continue-on-error: true
```
Comment (kkania-splunk) [inline .github/workflows/reusable-build-test-release.yml:208]: Im wondering if we need this flag here

**Review by kkania-splunk — APPROVED** [review]

File: .github/workflows/reusable-build-test-release.yml (hunk 1 above)
Comment (uoboda-splunk) [reply .github/workflows/reusable-build-test-release.yml:208]: Yes it will stop workflow if this fails

Comment (uoboda-splunk) [reply .github/workflows/reusable-build-test-release.yml:208]: And we want to progress anyway

**Pull request merged**